Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -1772,34 +1773,84 @@ private IQueryable<BaseItemEntity> TranslateQuery(
 
         if (filter.HasParentalRating ?? false)
         {
-            if (filter.MinParentalRating.HasValue)
+            var min = filter.MinParentalRating;
+            if (min != null)
             {
-                baseQuery = baseQuery
-                   .Where(e => e.InheritedParentalRatingValue >= filter.MinParentalRating.Value);
+                if (min.SubScore != null)
+                {
+                    baseQuery = baseQuery
+                        .Where(e => e.InheritedParentalRatingValue >= min.Score && e.InheritedParentalRatingSubValue >= min.SubScore);
```

Comment:
Wouldn't this always filter out anything in the database that has a `InheritedParentalRatingSubValue` of `null`?